File: src/main/java/de/tectoast/emolga/commands/music/control/QueueLoopCommand.java
package de.tectoast.emolga.commands.music.control;

import de.tectoast.emolga.commands.GuildCommandEvent;
import de.tectoast.emolga.commands.MusicCommand;
import de.tectoast.emolga.utils.music.GuildMusicManager;

public class QueueLoopCommand extends MusicCommand {

    public QueueLoopCommand() {
        super("queueloop", "Loopt die derzeitige Queue oder beendet die Loop");
        setArgumentTemplate(ArgumentManagerTemplate.noArgs());
    }

    @Override
    public void process(GuildCommandEvent e) {
        GuildMusicManager manager = getGuildAudioPlayer(e.getGuild());
        e.reply("QueueLoop wurde " + (manager.scheduler.toggleQueueLoop() ? "" : "de") + "aktiviert!");
    }
}

File: src/main/kotlin/de/tectoast/emolga/commands/music/control/QueueLoopCommand.kt
package de.tectoast.emolga.commands.music.control

import de.tectoast.emolga.commands.GuildCommandEvent
import de.tectoast.emolga.commands.MusicCommand

class QueueLoopCommand : MusicCommand("queueloop", "Loopt die derzeitige Queue oder beendet die Loop") {
    init {
        argumentTemplate = ArgumentManagerTemplate.noArgs()
    }

    override suspend fun process(e: GuildCommandEvent) {
        val manager = getGuildAudioPlayer(e.guild)
        e.reply("QueueLoop wurde " + (if (manager.scheduler.toggleQueueLoop()) "" else "de") + "aktiviert!")
    }
}